core/src/main/java/edu/wpi/grip/core/util/OpenCVUtility.java

```diff
@@ -0,0 +1,36 @@
+package edu.wpi.grip.core.util;
+
+
+import com.google.common.eventbus.EventBus;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_imgcodecs;
+
+import java.io.IOException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class OpenCVUtility {
```

Comment:
What if we want another utility method? Create another class for each utility method?